frontend/ui/elements/screen_notification_menu_table.lua

```diff
@@ -0,0 +1,184 @@
+local Notification = require("ui/widget/notification")
+local _ = require("gettext")
+
+local band = bit.band
+local bor = bit.bor
+local bxor = bit.bxor
+
+local function isMaskEnabled(source)
+    local mask = G_reader_settings:readSetting("notification_sources_to_show_mask") or Notification.SOURCE_DEFAULT
+    return band(mask, source) ~= 0
+end
+
+local function toggleMask(source)
+    local mask = G_reader_settings:readSetting("notification_sources_to_show_mask") or Notification.SOURCE_DEFAULT
+    G_reader_settings:saveSetting("notification_sources_to_show_mask", bxor(mask, source))
+end
+
+local function setMask(source)
+    G_reader_settings:saveSetting("notification_sources_to_show_mask", source)
+end
+
+local function getMask()
+    return G_reader_settings:readSetting("notification_sources_to_show_mask") or Notification.SOURCE_DEFAULT
+end
+
+local function showExpertMenu(toggle)
+    if toggle and toggle==true then
+        return {
+            text = _("Expert notification settings"),
+            help_text =_("Here you can set the sources of popup notifications on a fine grain scale."),
```

Comment:
Should we remove this ?
Its use is commented out below, but this still make these `_("")` translatable strings available to translators.

Reply:
Now only the `inflationary` menu is commented out. The submenu for single notifications is in.

But the question stays: Schould we comment the `Expert notification level` out, or leave it in?

Reply:
Well, that's mostly a developper option needed for you the last weeks to tweak this up :) It does not need to be a permanent help (I hope we're done with notification after this - or after you add a few notify() to the added dispatcher paging actions :)

So, either keep a backup of it on your side in case you need it again - or comment it out completely in the code (@Frenzie : can we trust the _() in comments won't be picked by weblate/gettext?)

Reply:
I don't see anything commented out in here? That means it definitely will be picked up. If you comment it out I think xgettext should skip it.

Reply:
I have deleted the submenu.